Etiquette/CODE_REVIEW.md

```diff
@@ -103,8 +103,8 @@ To improve collaboration, we take time to review pull request together in small
 Pull Assigners
 --------
 
-To balance the workload and avoid knowledge silos we use [Pull Assigners](https://pullpanda.com/assigner) to assign reviewers.
+To balance the workload and avoid knowledge silos we use GitHub's code reviewer assignment feature to automatically assign reviewers.
 
-- Each pull request will be assigned to three random reviewers from the team (PS: if the PullAssigners bot seems stuck, try to de-assign it and re-assign it.)
+- Each pull request will be assigned to three random reviewers from the team.
```

Comment:
Is it really random though? Might be interesting (and a bit reassuring) to mention that it's more like round-robin to ensure fair balance

Reply:
I've updated this to mention the routing algo. Let me know if it makes sense.